lms/static/js/learner_dashboard/views/unenroll_view.js

```diff
@@ -75,6 +75,11 @@ class UnenrollView extends Backbone.View {
         this.switchToSlideOne();
         this.$('.reasons_survey:first .submit_reasons').click(this.switchToSlideTwo.bind(this));
       }
+    } else if (xhr.status === 400) {
+      $('#unenroll_error').text(
+        gettext(xhr.responseText),
```

Comment:
I don't think you need this `gettext` call - any error should be translated already coming from the backend.

Reply:
Great! Thanks for the suggestion! I'm gonna change it